src/apps/authentication/admin.py

```diff
@@ -1,2 +1,9 @@
-
 # Register your models here.
+
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import User
+
+admin.site.register(User, UserAdmin)
```

Comment:
For some reason the User.is_verified is not manageable through django admin page

Reply:
It is done